README.md

```diff
@@ -156,9 +156,12 @@ There are two significant differences on EKS:
 ### Installing from a container
 
 This command copies the kube-bench binary and configuration files to your host from the Docker container:
-** binaries compiled for linux-x86-64 only (so they won't run on macOS or Windows) **
+> **Note:**
+> - Binaries compiled for linux-x86-64 only (so they won't run on macOS or Windows).
+> - To install, the container must be run as root user.
+
 ```
-docker run --rm -v `pwd`:/host aquasec/kube-bench:latest install
+docker run --rm -v `pwd`:/host -u 0:0 aquasec/kube-bench:latest install
```

Comment:
As above, the -u 0:0 option explicitly runs the container as root!

Reply:
This is necessary only to install the binary on the host. If it is run within the container with host mounted volumes, then it can run with non-root privileges.